<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu Laporan</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #E0E0E0;
    font-family: &quot;Arial&quot;;
}

QGroupBox {
    border: 2px solid #A0A0A0;
    border-radius: 5px;
    margin-top: 20px;
    background-color: #ECECEC;
    font-weight: bold;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 5px;
    left: 10px;
    color: #333;
}

/* Label Input Warna Kuning Krem seperti Jurnal */
QLabel#label_Mulai, QLabel#label_Sampai {
    background-color: #FFFACD;
    border: 1px solid #888888;
    padding: 5px;
    font-weight: bold;
    color: black;
}

/* Judul Besar */
QLabel#label_Judul {
    font-size: 24px;
    font-weight: bold;
    font-style: italic;
    color: #000000;
    margin-bottom: 10px;
}

/* Input Tanggal */
QDateEdit {
    background-color: white;
    border: 1px solid #888888;
    padding: 5px;
    color: black;
}

/* Tabel */
QTableWidget {
    background-color: white;
    border: 1px solid #888888;
    gridline-color: #CCCCCC;
    color: black;
}
QHeaderView::section {
    background-color: #D3D3D3;
    padding: 4px;
    border: 1px solid #888888;
    font-weight: bold;
}

/* Tombol Klasik */
QPushButton {
    background-color: #F5F5F5;
    border: 1px solid #888888;
    border-radius: 3px;
    padding: 10px;
    font-weight: bold;
    font-size: 12px;
    color: black;
}
QPushButton:hover {
    background-color: #FFFFFF;
}
QPushButton:pressed {
    background-color: #CCCCCC;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>:: LAPORAN ::</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_Judul">
        <property name="text">
         <string>Lihat Laporan</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="frameShadow">
         <enum>QFrame::Shadow::Sunken</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="verticalSpacing">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_Mulai">
          <property name="text">
           <string>Mulai Tanggal</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="dateEdit_Mulai">
          <property name="displayFormat">
           <string>yyyy-MM-dd</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_Sampai">
          <property name="text">
           <string>Sampai Dengan</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDateEdit" name="dateEdit_Sampai">
          <property name="displayFormat">
           <string>yyyy-MM-dd</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>Data Preview</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_LapPeminjaman">
        <property name="text">
         <string>Laporan Peminjaman</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_LapPembayaran">
        <property name="text">
         <string>Laporan Pembayaran</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_LapCetak">
        <property name="text">
         <string>Cetak Laporan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Keluar">
        <property name="text">
         <string>Keluar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
